stackit/internal/services/objectstorage/objectstorage_acc_test.go

```diff
@@ -168,14 +168,6 @@ func TestAccObjectStorageResource(t *testing.T) {
 						"stackit_objectstorage_credential.credential", "credential_id",
 						"data.stackit_objectstorage_credential.credential", "credential_id",
 					),
-					resource.TestCheckResourceAttrPair(
-						"stackit_objectstorage_credential.credential", "access_key",
-						"data.stackit_objectstorage_credential.credential", "access_key",
-					),
-					resource.TestCheckResourceAttrPair(
-						"stackit_objectstorage_credential.credential", "secret_access_key",
-						"data.stackit_objectstorage_credential.credential", "secret_access_key",
-					),
```

Comment:
What was happening with these two?

Reply:
Was failing because the data source doesn't have these keys, because they're only given on creation